### PR TITLE
lots of tweaks

### PR DESCRIPTION
**Pull request by ara4n**
* reword the abstract to better justify the existence of LM
* emphasise the importance of following authentication rules as per the room version
* clarify (i think) how DAG compatibility would work by "trunking" LM via a DAG-capable server.

### CODE REVIEW

File: draft-ralston-mimi-linearized-matrix.md [hunk 1]
```diff
@@ -88,16 +92,18 @@ informative:
...
-of the European Union's Digital Markets Act (DMA).
+This model provides excellent decentralization characteristics and conversation history
+replication, but proves complex when aiming to use Matrix strictly for interoperability
+between today's existing messaging service providers, which often do not persist
+chat history serverside, and do not seek to replicate it between servers.
 
-This document explores an API surface for Matrix which knowingly trades some of the
-decentralization aspects for ease of interoperability at a per-room level. We call this
-API surface "Linearized Matrix".
+This document explores an API surface for Matrix which optimises for ease of
```
Comment (turt2live) [inline draft-ralston-mimi-linearized-matrix.md]:
```suggestion
This document explores an API surface for Matrix which optimizes for ease of
```

File: draft-ralston-mimi-linearized-matrix.md [hunk 2]
```diff
@@ -88,16 +92,18 @@ informative:
...
+interoperability at the expense of decentralised conversation history at a per-room level.
```
Comment (turt2live) [inline draft-ralston-mimi-linearized-matrix.md]:
```suggestion
interoperability at the expense of decentralized conversation history at a per-room level.
```

File: draft-ralston-mimi-linearized-matrix.md [hunk 3]
```diff
@@ -220,6 +229,10 @@ A transport layer would describe a formal request/response structure for the mem
 requests should be able to be rejected by the owner server prior to the membership transition happening
 in the room, as the owner server may wish to apply additional checks for anti-abuse or similar.
 
+**XXX**: I still don't understand why we need membership APIs, rather than just sending the right state
+event to the server to invite/ban/kick/etc.  It just makes them look special-cased and makes the
+API look more bigger than it really is... --Matthew
```
Comment (turt2live) [inline draft-ralston-mimi-linearized-matrix.md]: Membership has additional context above normal state handling. 

We're not saying here that one needs to have a transport layer difference between membership and state, just calling out that membership is slightly weird.

File: draft-ralston-mimi-linearized-matrix.md (hunk 2 above)
Comment (turt2live) [reply draft-ralston-mimi-linearized-matrix.md]: ize throughout

File: draft-ralston-mimi-linearized-matrix.md [hunk 4]
```diff
@@ -289,10 +304,18 @@ Room Version 10 definition {{MxRoomVersion10}}. Changes will be made to support
...
 
-Linearized Matrix is simply a simpler API for accessing a room on Matrix, which means servers which support
-a full-blown DAG can still join and participate in the room. With DAG-compatible servers in the room, the
-DAG-compatible servers talk to each other directly as they do with the current Matrix APIs, only involving
-the room owner for state changes (like membership).
+Linearized Matrix is essentially an alternative API for accessing normal Matrix rooms over federation,
+which means servers which support a full-blown DAG can still join and participate in the room.
+This is critical in order to avoid breaking compatibility with today's fully-decentalised Matrix, and
+provides a way to decentralise ownership of rooms even if large messaging providers are themselves
+not able to implement full decentralisation yet (c.f. {{?I-D.avoiding-internet-centralization}}).
```
Comment (turt2live) [inline draft-ralston-mimi-linearized-matrix.md]:
```suggestion
not able to implement full decentralisation yet. {{?I-D.avoiding-internet-centralization}}
```

File: draft-ralston-mimi-linearized-matrix.md (hunk 3 above)
Comment (turt2live) [reply draft-ralston-mimi-linearized-matrix.md]: We can rename the section if needed.

Comment (ara4n) [reply draft-ralston-mimi-linearized-matrix.md]: okay, that works for me.

**Pull request merged**